packages/wonder-blocks-dropdown/components/action-menu.js

```diff
@@ -234,9 +234,10 @@ export default class ActionMenu extends React.Component<MenuProps, State> {
                 alignment={alignment}
                 items={menuItems}
                 light={false}
-                onClose={() => this.handleClose()}
+                onDropdownChanged={(open) => this.handleDropdownChanged(open)}
```

Comment:
`onDropdownChanged` feels a little too general as the contents of the what are in the dropdown don't change.  Maybe change this to `onOpenChanged` since that's exactly what's changing.